### ISSUE TITLE
Webextension marked as obsolete in Firefox 55 !?

### ISSUE THREAD

**Issue by seboss666** [sarahavilov/secure-login#41]
Hello,

I have a strange behaviour from Firefox, that report "obsolete" even though I have the webextensions flavour of Secure Login (I switch almost directly after the announcement of the availability). I'm on Manjaro Linux, with the latest stable version (55.0.1 64bit).

What could go wrong ? May an old reference in some of the profile's databases be misleading FF ?
![obsolete](https://user-images.githubusercontent.com/2640124/29483905-01b5274c-84b3-11e7-9fa9-36e5bac91ff8.png)

It may be related to https://github.com/sarahavilov/secure-login/issues/40 

(anyway, I use this thread to thank you for your work on it :) )

To be sure I'm not encountering influence from other extensions, I'm looking to replace all the others in the capture picture, I may update when it's done.

**Comment by d4k0** [comment]
I think Secure Login isn't a "real" WebExtension add-on yet, but a WebExtension add-on embedded in one of the old add-on types (the name also suggests this). uBlock Origin does the same as far as I know, but there is a "real" WebExtension version of it for testing which will replace the old one when Firefox 57 is released.

**Comment by sarahavilov** [comment]
as @d4k0 mentioned, SL is an embedded WE. Unfortunetly the API to retrieve credentials is not yet implemented and hence there is no way to release the actual WE extension at this point!

**Comment by crssi** [comment]
@sarahavilov is there any mozilla plan for needed API?
If it is, do you know for which version of FF is scheduled?

Anyway, very good job and thank you for this WE. It is one of 16 addons/WE I use and "I could not live without" :)

cheers

**Comment by d4k0** [comment]
@crssi, I found this:

https://bugzilla.mozilla.org/show_bug.cgi?id=1285270
https://bugzilla.mozilla.org/show_bug.cgi?id=1324919
https://github.com/web-ext-experiments/logins

It seems that the API won't be ready for Firefox 57 :-(.

**Comment by crssi** [comment]
crap :(
There is so many issues open for WEs that I hope mozilla will pospone addons blockage to FF58, or at least until they resolve those issues.

**Comment by sarahavilov** [comment]
Since it is very unlikely Mozilla provides some useful APIs soon, you can consider using KeePass application. If so switch to https://addons.mozilla.org/en-US/firefox/addon/keepasshelper/

**Comment by crssi** [comment]
Nah, keepass is no match to your extension, where I can have firefox sync between computers without complicate installing keepass everywhere i need. ;)
Will have to click manually until mozilla resolves showstopper for your great extension or will stay on FF55, 56 or ESR until resolved.
Thank you

**Comment by sarahavilov** [comment]
Definitely the internal password manager is easier to use. But regarding KeePass, you can share the database with Dropbox and use it in multiple PCs. At least this is what I am planning to do.

**Comment by WyattBest** [comment]
I'm also just manually clicking until this is available in FF57+. This was one of my top five extensions!

**Comment by crssi** [comment]
^^Same here.
It looks we will manually click at least until FF59 :( [See here](https://bugzilla.mozilla.org/show_bug.cgi?id=1324919)
